### ISSUE TITLE
Set default font to Cascadia Mono on Windows (or any font that displays unicode properly)

### ISSUE THREAD

**Issue by expikr** [JuliaLang/juliaup#642]
See https://github.com/fonsp/Pluto.jl/issues/2445

The repl executable on Windows should set its conhost instance to use the Cascadia Mono font, so that it works with UnicodePlots out of the box.

Like so:

![image](https://github.com/JuliaLang/juliaup/assets/77922942/13122ece-2712-438a-adac-704ec283087c)

**Comment by davidanthoff** [comment]
I don't think we can do that, we use an MSIX installer and I can't find anything in there how we could configure that.
